x-pack/plugin/ml/src/main/plugin-metadata/entitlement-policy.yaml

```diff
@@ -0,0 +1,4 @@
+org.elasticsearch.ml:
+  - files:
+      - path: "tmp/"
```

Comment:
This is a relative path, relative to the ES installation? What is ML reading from here, that it isn't writing?

Reply:
Is it this?
> [class org.elasticsearch.xpack.ml.utils.NamedPipeHelper$PrivilegedInputPipeOpener], entitlement [file], operation [read], path [/dev/shm/bk/bk-agent-prod-gcp-1739377746684671266/elastic/elasticsearch-pull-request/x-pack/plugin/ent-search/build/testclusters/javaRestTest-0/tmp/controller_log_41038]

That should be the ES tmp dir, which everything will soon have access to (separate PR incoming).

Reply:
https://github.com/elastic/elasticsearch/pull/122370